sw/host/opentitantool/src/command/otp.rs

```diff
@@ -0,0 +1,112 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::any::Any;
+use std::fs::File;
+use std::io::Write;
+use std::path::PathBuf;
+
+use anyhow::{bail, Result};
+
+use serde_annotate::{serialize, Annotate, Base};
+
+use structopt::StructOpt;
+
+use opentitanlib::app::command::CommandDispatch;
+use opentitanlib::app::TransportWrapper;
+use opentitanlib::otp::alert_handler::AlertRegs;
+use opentitanlib::otp::lc_state::LcStateVal;
+use opentitanlib::otp::otp_img::{OtpImg, OtpImgItem, OtpImgPartition, OtpImgValue};
+
+/// Generate CRC magic value for alert_handler configuration.
+#[derive(Debug, StructOpt)]
+pub struct AlertDigest {
+    #[structopt(help = "OTP memory map file containing alert_handler config in HJSON format")]
+    alert_cfg: PathBuf,
+    #[structopt(help = r#"Life cycle state ("TEST", "DEV", "PROD", PROD_END, "RMA")"#)]
+    lc_state: String,
+    #[structopt(
+        long,
+        help = "Output file to write the new OTP overlay to instead of printing"
+    )]
+    output: Option<PathBuf>,
+    #[structopt(
+        long,
+        default_value = "OWNER_SW_CFG",
+        help = "Override switch to specify a custom partition"
+    )]
+    partition: String,
+    #[structopt(long, help = "Override switch to specify a custom OTP item name")]
+    name: Option<String>,
```

Comment:
I think we can remove these flags if we don't plan to use them.

Reply:
I wasn't sure. I didn't want to assume the names of the OTP words would have the same format, so I added this in case they needed to be overwritten from the bazel side of things. I figured it's an optional switch, so if it's not necessary in the worst case it will show up when you call `opentitantool otp alert-digest --help`.